capstone/src/main/resources/static/user.js

```diff
@@ -172,38 +132,28 @@ function hideFields(selectField, universityField) {
 
 /* get the saved events for individual users */
 function getIndividualEvents() {
-  var userType = sessionStorage.getItem("user-type");
-  if (userType == null) {
-    getUserType(true);
-  }
-  if(userType == "individual") {
-    fetch('get-' + userType).then(response => response.json()).then((data) => {
+  fetch('user-info').then(response => response.json()).then((data) => {
+    if(data.userType == "individual") {
       const eventDiv = document.getElementById("saved-events");
-      data[0].savedEvents.forEach((event) => eventDiv.appendChild(createSavedEventElement(event, false, false)));
+      data.savedEvents.forEach((event) => eventDiv.appendChild(createSavedEventElement(event, false, false)));
       displayMain(true);
-    });
-  } else {
-    displayMain(false);
-  }
-
+    } else {
+      displayMain(false);
+    }
+  });
 }
 
-/* get the saved organizations for individual users */
+/* Function to get the saved organizations */
 function getIndividualOrganizations() {
-  var userType = sessionStorage.getItem("user-type");
-  if (userType == null) {
-    getUserType(true);
-  }
-  if (userType == "individual") {
-    fetch('get-' + userType).then(response => response.json()).then((data) => {
+  fetch('user-info').then(response => response.json()).then((data) => {
+    if(data.userType == "individual") {
```

Comment:
You know how we usually ask for more helper functions? In this case, I think the opposite could be helpful -- because `getIndividualEvents()` and `getIndividualOrganizations()` are really similar, perhaps they could merge to become a single helper method.